src/components/client/core/Tooltip/Tooltip.tsx

```diff
@@ -17,6 +17,7 @@ import type { ReactNode } from "react";
 export interface Props extends TooltipProps {
   trigger: ReactNode;
   content: ReactNode;
+  bgColor?: string;
```

Comment:
The `bgColor` type should be `JsxStyleProps["bgColor"]` I believe. AFK right now, but the `JsxStyleProps` should be imported from `generated/panda/jsx` iirc. Will test everything when I have the chance in a bit on my laptop!